pkg/util/timeutil/time_zone_util.go

```diff
@@ -17,22 +17,37 @@ import (
 	"strings"
 	"time"
 
+	"github.com/cockroachdb/cockroach/pkg/sql/pgwire/pgcode"
+	"github.com/cockroachdb/cockroach/pkg/sql/pgwire/pgerror"
 	"github.com/cockroachdb/errors"
 )
 
 const (
-	fixedOffsetPrefix = "fixed offset:"
-	offsetBoundSecs   = 167*60*60 + 59*60
+	offsetBoundSecs = 167*60*60 + 59*60
+	// PG supports UTC hour offsets in the range [-167, 167].
+	maxUTCHourOffset = 167
 )
 
 var timezoneOffsetRegex = regexp.MustCompile(`(?i)^(GMT|UTC)?([+-])?(\d{1,3}(:[0-5]?\d){0,2})$`)
 
-// FixedOffsetTimeZoneToLocation creates a time.Location with a set offset and
-// with a name that can be marshaled by crdb between nodes.
-func FixedOffsetTimeZoneToLocation(offset int, origRepr string) *time.Location {
-	return time.FixedZone(
-		fmt.Sprintf("%s%d (%s)", fixedOffsetPrefix, offset, origRepr),
-		offset)
+// FixedTimeZoneStringToLocation creates a time.Location with an offset and a
+// time zone string.
+func FixedTimeZoneStringToLocation(offset int, origRepr string) *time.Location {
```

Comment:
`FixedTimeZoneOffset` maybe? `StringToLocation` doesn't make sense to me.

Reply:
Renamed